README.md

```diff
@@ -1,18 +1,648 @@
 
-For more information read [README_VERITABLE.md](./README_VERITABLE.md)
 
-## Configuration
+## Setup
+The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI (http://localhost:3000/swagger) that is provided with the server. The OpenAPI spec can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
 
-`veritable-cloudagent` takes the following arguments to configure the `AFJ Rest agent`:
+In case `npm i` fails with the references to the `node-gyp`. Please install xcode utilities by ruuning -> `brew install xcode`. Before executing `npm i` delete your node_modules in case it has some old references `rm -rf /node_modules`
 
-| variable      | required |        default         | description                                                         |
-| :------------ | :------: | :--------------------: | :------------------------------------------------------------------ |
-| persona-title |    N     | `Veritable Cloudagent` | Site title for the service's OpenAPI documentation                  |
-| persona-color |    N     |        `white`         | CSS colour value for the service's OpenAPI documentation background |
+#### DRPC Config
+The RPC client can be configured through Envs, and follows the following format:
+
+```
+{
+  ...,
+  "verifiedDrpcOptions": {
+    "proofTimeoutMs": <int> (timeout on proof requests, default: 5000),
+    "requestTimeoutMs": <int> (timeout on DRPC requests, default: 5000),
+    "credDefId": <string>, (credential definition ID to add to restrictions, can be easily set through "VERIFIED_DRPC_OPTIONS_CRED_DEF_ID" env)
+    "proofRequestOptions": (proof options) <CreateProofRequestOptions<ProofProtocol[]>>
+    }
+  },
+  ...
+}
+```
+
+Note that the proof options must be set through envs due to the fact that a DRPC request handler is configured during initialisation.
+
+### Prerequisites
+- postgres:16.3+
+- xcode utilities (node-gyp)
+- npm 10.0.0+
+- node 20.0.0+
+
+## Getting started
+The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI (http://localhost:3000/swagger) that is provided with the server. The OpenAPI spec can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
+
+Bellow you will find commands for starting up the containers in docker (see the [Using Docker](#using-docker-easiest) section).
+
+> The OpenAPI spec is generated from the model classes used by Aries Framework JavaScript. Due to limitations in the inspection of these classes, the generated schema does not always exactly match the expected format. Keep this in mind when using this package. If you encounter any issues, feel free to open an issue.
+
+> In case `npm i` fails with the references to the `node-gyp`. Please install xcode utilities by ruuning -> `brew install xcode`. Before executing `npm i` delete your node_modules in case it has some old references `rm -rf /node_modules`
```

Comment:
`rUUning`

Reply:
🤦 was too many moving parts so eyes did not even notice